crates/router/src/core/payments/transformers.rs

```diff
@@ -52,6 +53,10 @@ where
     )
     .await?;
 
+    fp_utils::when(merchant_connector_account.is_disabled(), || {
+        Err(errors::ApiErrorResponse::MerchantConnectorAccountDisabled)
```

Comment:
Can you add the connector label here? It will be more clear as to which mca was disabled.

Reply:
I agree, but the type is a abstraction over db value and cache value. And getting to look inside, the cache value doesn't provide this label.

Reply:
You already have the connector label that you are querying with, here https://github.com/juspay/hyperswitch/pull/1141/files#diff-6f19e8956b6645147dedca51bdadb551b2cfbe7ce0a7092e514a4dfa41deb3bcR51 Use that